docs/platform/api/getting-started-with-the-linode-api-v4.md

```diff
@@ -0,0 +1,358 @@
+          },
+          ...
+      ],
+      "pages": 1,
+      "results": 18
+  }
+{{< /file-excerpt >}}
+
+API responses will contain either a list or an object, depending on the request. Lists are paginated and contain the following information:
+
+  * `pages`: Total number of pages
+  * `results`: Total number of results
+  * `page`: Current page
+  * `data`: Response body
+
+  In general, queries to higher-level endpoints will return a list, and more specific queries will return an object. For example, the query to `/images` above returned a list of all available images. To get information about a specific image, you can query that image directly using the format `/images/id` (the ID can be found in the `id` field of the response above). This query will return a single object:
+
+    curl https://api.linode.com/v4/images/linode/slackware13.37
+
+  {{< file-excerpt "response.json" >}}
+{
+    "deprecated": true,
+    "created_by": "linode",
+    "id": "linode/slackware13.37",
+    "created": "2011-06-05T19:11:59",
+    "is_public": true,
+    "description": null,
+    "size": 600,
+    "label": "Slackware 13.37",
+    "type": "manual",
+    "vendor": "Slackware"
+}
+{{< /file-excerpt >}}
+
+### Pagination
+
+If a results list contains more than 100 items, the response will be split into multiple pages. Each response will include the total number of pages and the current page. To view additional pages, add a `page` parameter to the end of the URL. For example, querying the available kernels produces more than 200 results:
+
+
+    curl https://api.linode.com/v4/linode/kernels
+
+
+  {{< file-excerpt "results.json" json >}}
+{
+    "results": 214,
+    "page": 1,
+    "data": [
+        {
+            "kvm": false,
+            "architecture": "i386",
+            "version": "2.6.28",
+            "xen": true,
+            "label": "2.6.28.3-linode17",
+            "id": "linode/2.6.28.3-linode17",
+            "pvops": true
+        },
+        {
+            "kvm": false,
+            "architecture": "i386",
+            "version": "2.6.18",
+            "xen": true,
+            "label": "2.6.18.8-linode16",
+            "id": "linode/2.6.18.8-linode16",
+            "pvops": false
+        },
+    ]
+        ...
+    "pages": 3
+}
+{{< /file-excerpt >}}
+
+From the response you can see that the results are divided into three pages. View the second page:
+
+    curl https://api.linode.com/v4/linode/kernels?page=2
+
+If you would prefer a smaller number of items per page, you can override the default value with the `page_size` parameter:
+
+    curl https://api.linode.com/v4/linode/kernels?page_size=50
+
+### Filter Results
+
+Version 4 also supports filtering request results. Filters are passed using the `X-Filter` header and use JSON format. You can filter on almost any field that appears in a response object. The following query uses the `deprecated` and `vendor` fields to return all current Debian images:
```

Comment:
Not sure if it's worth noting, but the api docs mark fields that are filterable.